main.go

```diff
@@ -2,13 +2,38 @@ package main
 
 import (
 	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
 	"github.com/bwmarrin/discordgo"
+	"github.com/joho/godotenv"
 )
 
+var token string
+
+func init() {
+	if err := godotenv.Load(); err != nil {
```

Comment:
what would happen during deployment if/when this failed? I think we wouldn't be able to deploy unless there was a `.env` present in the filesystem.

Another approach I've seen is just using godotenv as a binary and not as a dependency of the project since it's a dev tool. The way we run the project locally would then be:
```sh
godotenv -f .env go run main.go
```

Reply:
Nice! Sounds good! I am no longer loading the .env file at run time. Updated the readme with this command.